apps/intellij/src/main/kotlin/dev/nx/console/nxls/NxlsProcess.kt

```diff
@@ -58,7 +57,7 @@ class NxlsProcess(private val project: Project, private val cs: CoroutineScope)
 
                             logger.trace("Nxls early exit: $this")
 
-                            Notifier.notifyNxlsError(project)
+                            //                            Notifier.notifyNxlsError(project)
```

Comment:
We don't want this notifier?

Reply:
No, the exit is now triggered in "allowed" circumstances when restarting. We will need to provide a message when the nxls errors out in the future, but this was being triggered wrongly.

Reply:
Let's remove it then